src/sparseml/transformers/base.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from typing import Optional
+
+from sparseml.base import check_version
+
+
+_LOGGER = logging.getLogger(__name__)
+
+_TRANSFORMERS_MIN_VERSION = "4.36"
```

Comment:
two things
1. let's warn instead of raise here, having only 1 minor version supported is too restrictive
2. should support an API or environment variable (or both) to skip the modification step